modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2763,7 +2763,14 @@ private GroupByStep _getOneToManyObjectEntriesGroupByStep(
 									objectField.getDBColumnName());
 					}
 
-					return column.eq(related ? primaryKey : 0L);
+					if (related) {
+						return column.eq(primaryKey);
+					}
+
+					return column.isNull(
+					).or(
+						column.eq(0L)
+					).withParentheses();
```

Comment:
Hi @victorhcunha, maybe you included this commit by mistake

Reply:
Yes! My mistake, sorry.